es_mapper.yaml

```diff
@@ -104,65 +112,64 @@ target-mappings:
                       exclude:
                       - _links
                       - participant_id
+                      - created_at
+                      - modified_at
                       properties: {}
                     outcome:
                       source: Outcome
                       exclude:
                       - _links
+                      - created_at
+                      - modified_at
                       properties: {}
                     phenotype:
                       properties:
                         hpo:
                           source: Phenotype
                           exclude:
                           - _links
-                          - hpo_id
+                          - created_at
+                          - modified_at
+                          - hpo_id_phenotype
                           - participant_id
-                          - snomed_id
+                          - observed
                           - kf_id
                           properties:
                             ancestral_hpo_ids:
                               type: keyword
-                            negative_hpo_ids:
+                            phenotype_not_observed:
                               type: keyword
-                            hpo_ids:
+                            phenotype_observed:
```

Comment:
Does `phenotype_observed` refer to `hpo_id_phenotype` field in data service? If that's the case, should it be better to name the field as `hpo_phenotype_observed`? Particularly consider that there is another phenotype field: `snomed_id_phenotype` (possibly new ones later).

Same for `phenotype_not_observed` field, changing it to `hpo_phenotype_not_observed`?

Reply:
@junjun-zhang Addresses `snomed_phenotype_observed` and `snomed_phenotype_not_observed`